Console/Command/QueueShell.php

```diff
@@ -291,7 +413,7 @@ protected function _listTasks() {
 		}
 	}
 	
-	function out($str='') {
+	function out($str = null, $newlines = 1, $level = Shell::NORMAL) {
```

Comment:
I tried it, it broke the output for some reason (it was doubling the time stamp.)  I took out the passing to the parent.